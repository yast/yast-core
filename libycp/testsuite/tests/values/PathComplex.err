Parsed:
----------------------------------------------------------------------
.""
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
."habakuk"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
."!@#$%^&*()+_-{}[]|~`,.<>/?;:'"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
."-"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
."-text-"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
."\"\\\n\t\r\b \f A ..."
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
."\"\"\""
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
."a\"a\"a"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
."X41000"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
."MNÃ¡"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
."".""
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
."habakuk"."habakuk"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
."-"."-"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
."-text-"."-text-"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
."\"\\\n\t\r\b \f A ..."."\"\\\n\t\r\b \f A ..."
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
."\"\"\""."\"\"\""
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
."a\"a\"a"."a\"a\"a"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(. + ."example"."path")
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(."com po sed" + ."pa th")
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
((.three.times + ."\"concatenated\"") + ."!@#$%^")
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(((.obscur + .example) + ."") + ."--")
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(((.obscur + .example) + .) + ."")
----------------------------------------------------------------------
