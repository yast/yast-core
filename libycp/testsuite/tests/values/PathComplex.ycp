
# ---------------------------------------------------------
#
#  Filename:    PathComplex.ycp
#
#  Purpose:     test cases for YCP test. Test path with
#               sections between " "
#
#  Creator:     pblahos@suse.cz
#
#  Maintainer:  pblahos@suse.cz
#
# ---------------------------------------------------------


// this tests includes tests for YCP_Path

# ------- YCP_Path
   // strange but allowed (p = .; is not allowed)
.""
."habakuk"
."!@#$%^&*()+_-{}[]|~`,.<>/?;:'"
."-"
."-text-"
."\"\\\n\t\r\b \f \x41 ..."
."\"\"\""
."a\"a\"a"
."\X41\000"
."\x4D\x4e\xc3\xa1"
."".""
."habakuk"."habakuk"
."-"."-"
."-text-"."-text-"
."\"\\\n\t\r\b \f \x41 ..."."\"\\\n\t\r\b \f \x41 ..."
."\"\"\""."\"\"\""
."a\"a\"a"."a\"a\"a"

# ------- concatenating paths
( . + ."example"."path" )
( ."com po sed" + ."pa th" )
( .three.times + ."\"concatenated\"" + ."!@#$%^" )
( .obscur + .example + ."" + ."--" )
(.obscur+.example+.+."")
