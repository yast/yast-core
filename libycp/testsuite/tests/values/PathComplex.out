(."")
(."habakuk")
(."!@#$%^&*()+_-{}[]|~`,.<>/?;:'")
(."-")
(."-text-")
(."\"\\\n\t\r\b \f A ...")
(."\"\"\"")
(."a\"a\"a")
(."X41000")
(."MNÃ¡")
(.""."")
(."habakuk"."habakuk")
(."-"."-")
(."-text-"."-text-")
(."\"\\\n\t\r\b \f A ..."."\"\\\n\t\r\b \f A ...")
(."\"\"\""."\"\"\"")
(."a\"a\"a"."a\"a\"a")
(."example"."path")
(."com po sed"."pa th")
(.three.times."\"concatenated\""."!@#$%^")
(.obscur.example.""."--")
(.obscur.example."")
